### PR TITLE
Editor revision for TC meeting 2024-10-30

### PR DESCRIPTION
**Pull request by tschmidtb51**
- fixes https://github.com/oasis-tcs/csaf/issues/796
- fixes https://github.com/oasis-tcs/csaf/issues/785
- fixes https://github.com/oasis-tcs/csaf/issues/798
- fixes https://github.com/oasis-tcs/csaf/issues/563
- fixes https://github.com/oasis-tcs/csaf/issues/662
- fixes https://github.com/oasis-tcs/csaf/issues/730
- fixes https://github.com/oasis-tcs/csaf/issues/541
- fixes https://github.com/oasis-tcs/csaf/issues/633
- revert #788 
- addresses parts of https://github.com/oasis-tcs/csaf/issues/469
- fixes https://github.com/oasis-tcs/csaf/issues/811
- addresses parts of https://github.com/oasis-tcs/csaf/issues/318
- fixes https://github.com/oasis-tcs/csaf/issues/674

### CODE REVIEW

File: csaf_2.1/prose/edit/src/conformance.md [hunk 1]
```diff
@@ -627,4 +663,14 @@ A CSAF library satisfies the "CSAF library with extended validation" conformance
...
+### Conformance Clause 23: CSAF downloader
+
+A program satisfies the "CSAF downloader" conformance profile if the program:
+
+* conforms to the process defined in section [sec](#retrieving-rules) by executing all parts that are applicable to the given role.
+* supports directory-based and ROLIE-based retrieval.
+* is able to execute both steps from section [sec](#retrieving-rules) separately.
+* uses a program-specific HTTP User Agent, e.g. consisting of the name and version of the program.
+
+> A tool MAY implement an option to store CSAF documents that fail any of the steps in section [sec](#retrieving-csaf-documents)
```
Comment (tschmidtb51) [inline csaf_2.1/prose/edit/src/conformance.md:675]:
```suggestion
> A tool MAY implement an option to store CSAF documents that fail any of the steps in section [sec](#retrieving-csaf-documents).

**Review by sthagen — APPROVED** [review]
LGTM.

**Comment by tschmidtb51** [comment]
Motion set by Thomas Schmidt and seconded by Justin Murphy: https://groups.oasis-open.org/discussion/motion-for-819

**Review by sthagen — APPROVED** [review]
LGTM.

**Comment by tschmidtb51** [comment]
No objections or discussions received. Motion therefore passed.

**Comment by santosomar** [comment]
The motion to accept the Editor Revision 2024-10-30 as suggested in https://github.com/oasis-tcs/csaf/pull/819 into our master branch has passed.

**Pull request merged**